pycon.md

```diff
@@ -171,6 +182,8 @@ But it does not work beyond Python 3.9...
 
 (insert a lot of work here)
 
+TODO: ↑ is that a meta comment for you and me? 
+
```

Comment:
No, just hinting that a lot of things have been done. I might just remove it and only say it.